README.ios.md

```diff
@@ -2,7 +2,7 @@
 
 ## Prerequisites (Mac OS X)
 
-* Xcode 14 or higher
+* Xcode 14 or higher (and install platform iOS 17.5 SDK or higher)
```

Comment:
We have a minimum SDK of 12.0 for iOS. Specifying to install the latest iOS SDK should be sufficient.